### DISCUSSION TITLE
Disable Maven warnings reporting in Jenkins

### DISCUSSION THREAD

**Discussion by akurtakov** [eclipse-platform/eclipse.platform.ui#1644] [Polls]
This is continuation of https://github.com/eclipse-platform/eclipse.platform.ui/discussions/1567 which left room for interpretation thus this extra vote.
The vote will run for 7 days (results will be implemented based on status on Feb 13 Europe work hours)

**Comment by mickaelistria** [comment]
Reported as in generating a negative vote on GitHub, or reported as in visible in the logs?

**Reply by jukzi** [reply]
The maven warnings will allways be present in the jenkins job logs.
The question is if the mavenConsole() should filter the logs to show all ERROR, WARNING in the logs to github and fail the build.
with mavenConsole() enabled almost all PRs are marked as failed, as there are almost always some WARNINGs in the log (unrelated to the change in the PR). Also mavenConsole() duplicates WARNINGs from other plugins to be shown twice on jenkins. For example any random failed test not only shows as failed test but also as a "maven" ERROR.

**Reply by mickaelistria** [reply]
Thanks for the clarification.

**Comment by BeckerWdf** [comment]
I would expect a CI-Build for a PR to be green if now new issue (compilation issue, new warning, new test failures, new API issues, ...) where introduced with that PR.

**Reply by laeubi** [reply]
> I would expect a CI-Build for a PR to be green if now new issue (compilation issue, new warning, new test failures, new API issues, ...) where introduced with that PR.

So just assume the build does not gets executed at all due to an infra issue (--> now new issue (compilation issue, new warning, new test failures, new API issues, ...) where introduced **with that PR**) we should show it as a green successful build and merge any PR? 

Just wondering ;-P

**Reply by BeckerWdf** [reply]
@laeubi: Come on. You know what I mean...

**Comment by akurtakov** [comment]
Closing the vote. Winner is "keep reporting maven warnings".
